test/runtests.jl

```diff
@@ -251,6 +251,9 @@ end
 @test sprint(JSON.print, [NaN]) == "[null]"
 @test sprint(JSON.print, [Inf]) == "[null]"
 
+# check for issue #163
+@test isapprox(JSON.parse(json(Float32(2.1e-8))), 2.1e-8)
```

Comment:
Why are we testing `isapprox` here? Does converting to `Float64` first not roundtrip properly?